pkg/sentry/arch/signal_arm64.go

```diff
@@ -86,10 +85,6 @@ func (c *context64) NewSignalStack() NativeSignalStack {
 func (c *context64) SignalSetup(st *Stack, act *SignalAct, info *SignalInfo, alt *SignalStack, sigset linux.SignalSet) error {
 	sp := st.Bottom
 
-	if !(alt.IsEnabled() && sp == alt.Top()) {
-		sp -= 128
-	}
```

Comment:
Why do we not need this?

Reply:
Hi @avagin ,
According to the linux code and the abi spec, stack frame on Arm64 has no red zone.
Please see the following as reference:
x86 linux code:
https://github.com/torvalds/linux/blob/v5.10/arch/x86/kernel/signal.c#L243
Arm64 linux code:
https://github.com/torvalds/linux/blob/v5.10/arch/arm64/kernel/signal.c#L711

ABI Spec:
x86:
https://gitlab.com/x86-psABIs/x86-64-ABI/-/blob/master/x86-64-ABI/low-level-sys-info.tex#L498
Arm64 ABI:
https://github.com/ARM-software/abi-aa/blob/f52e1ad3f81254497a83578dc102f6aac89e52d0/aapcs64/aapcs64.rst#the-stack

According to my understanding, regarding to stack frame, for x86, it reuires:
  - end of input area is 16-byte aligned (asm)
  - check red zone is not destroyed by signal handlers
  for arm64, it requires:
  - end of input area is 16-byte aligned (asm)
  -